packages/api-specification/transform-type-info.js

```diff
@@ -94,72 +103,82 @@ const documentClass = (className) => {
-          h2('Methods'),
-          section(d.runner(jspath.apply('.{.kindString === "Method"}', d.match.children)))
-        ])
+          jspath.apply('.{.kindString === "Constructor"}', d.match.children).length !== 0 ? h3('Constructors') : undefined,
+          jspath.apply('.{.kindString === "Constructor"}', d.match.children).length !== 0 ? section(d.runner(jspath.apply('.{.kindString === "Constructor"}', d.match.children)), {class: 'constructors'}) : undefined,
+          jspath.apply('.{.kindString === "Property"}', d.match.children).length !== 0 ? h3('Properties') : undefined,
+          jspath.apply('.{.kindString === "Property"}', d.match.children).length !== 0 ? section(d.runner(jspath.apply('.{.kindString === "Property"}', d.match.children)), {class: 'properties'}) : undefined,
+          jspath.apply('.{.kindString === "Method"}', d.match.children).length !== 0 ? h3('Methods') : undefined,
+          jspath.apply('.{.kindString === "Method"}', d.match.children).length !== 0 ? section(d.runner(jspath.apply('.{.kindString === "Method"}', d.match.children)), {class: 'methods'}) : undefined
+        ], {id: className, class: 'jumptarget'})
     ),
     jsont.pathRule(
       '.{.kindString === "Method" || .kindString === "Constructor"}', d =>
         // we take just the first signature as the definition of the method, not
         // sure when there might be multiple signatures?!
         section([
-          d.runner(d.match.signatures[0]),
-          h4('test results'),
-          p(JSON.stringify(d.match.results))
-        ])
+          d.runner(d.match.signatures[0])
+        ], {class: 'method'})
     ),
     jsont.pathRule(
       '.{.kindString === "Property"}', d =>
         section([
-          h4(d.match.name),
+          h5(d.match.name, {class: 'code'}),
           p(d.match.comment ? d.match.comment.shortText : '')
-        ])
+        ], {class: 'property', id: d.match.name})
     ),
     jsont.pathRule(
       '.{.kindString === "Call signature" || .kindString === "Constructor signature"}', d =>
         section([
-          h3(d.match.name),
+          h4(d.match.name, {class: 'method-name', id: `${className}-${d.match.name}`}),
+          d.match.results ? section([
+            p('Electron'),
+            p(`${d.match.results.electron.passed}/${d.match.results.electron.total}`, {class: 'test-result ' + testResultPercentage(d.match.results.electron.passed, d.match.results.electron.total)}),
+            p('OpenFin'),
+            p(`${d.match.results.openfin.passed}/${d.match.results.openfin.total}`, {class: 'test-result ' + testResultPercentage(d.match.results.openfin.passed, d.match.results.openfin.total)}),
+            p('Browser'),
+            p(`${d.match.results.browser.passed}/${d.match.results.browser.total}`, {class: 'test-result ' + testResultPercentage(d.match.results.browser.passed, d.match.results.browser.total)})
```

Comment:
A few very minor points ...

- I don't think `p` tags make much sense here, how about `h5` for the container name, and a `span` for the results
- There's a bit of duplication on the 3 lines that output the results, might be worth moving into a separate function.